src/Stripe.net/Entities/Sources/StripeThreeDSecure.cs

```diff
@@ -4,13 +4,70 @@
 
     public class StripeThreeDSecure : StripeEntity
     {
+        [JsonProperty("address_line1_check")]
+        public string AddressLine1Check { get; set; }
+
+        [JsonProperty("address_zip_check")]
+        public string AddressZipCheck { get; set; }
+
+        [JsonProperty("authenticated")]
+        public bool Authenticated { get; set; }
+
+        [JsonProperty("brand")]
+        public string Brand { get; set; }
+
         [JsonProperty("card")]
         public string CardId { get; set; }
 
+        [JsonProperty("card_automatically_updated")]
+        public bool CardAutomaticallyUpdated { get; set; }
+
+        [JsonProperty("country")]
+        public string Country { get; set; }
+
         [JsonProperty("customer")]
         public string CustomerId { get; set; }
 
-        [JsonProperty("authenticated")]
-        public bool Authenticated { get; set; }
+        [JsonProperty("cvc_check")]
+        public string CvcCheck { get; set; }
+
+        [JsonProperty("description")]
+        public string Description { get; set; }
+
+        [JsonProperty("dynamic_last4")]
+        public string DynamicLast4 { get; set; }
+
+        [JsonProperty("exp_month")]
+        public int ExpMonth { get; set; }
+
+        [JsonProperty("exp_year")]
+        public int ExpYear { get; set; }
+
+        [JsonProperty("fingerprint")]
+        public string Fingerprint { get; set; }
+
+        [JsonProperty("funding")]
+        public string Funding { get; set; }
+
+        [JsonProperty("google_reference")]
+        public string GoogleReference { get; set; }
```

Comment:
I'm not sure what this is but it seems more internal than external I would not surface. I feel the same about IIN + Issuer + Description. It's gated and while it's supported on the Source resource I'm not convinced we should surface it here

Reply:
`IIN`, `Issuer` and `Description` are already available on `StripeCard` though.

Reply:
I removed `GoogleReference`.

Reply:
I know they are visible but I wanted to avoid leaking them further. Can you at least do what we do on the other resource: and group them with a comment, or add a comment to each one about being gated?